### PR TITLE
Have a RecordPutTransformer which supports null schema and use it in …

### PR DESCRIPTION
**Pull request by anwar6953**
Have a RecordPutTransformer which supports null schema and use it in TableSink, RealtimeTableSink and HBaseSink.

The RecordPutTransformer class has been copied over from the CDAP repo.

### CODE REVIEW

**Comment by gsps1** [comment]
fix checkstyle issues. changes lgtm, might be useful to copy the REcordputTrasnformerTest as well

**Comment by anwar6953** [comment]
Fixed the checkstyle issue. Since this class is copied directly from cdap, and is only temporary, I'll copy just this class.

**Comment by anwar6953** [comment]
RealtimeTableSinkTest is failing because one of the test cases expected a NPE when null schema is given.
https://travis-ci.org/caskdata/hydrator-plugins/builds/102738882

Updated it.

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java [hunk 1]
```diff
@@ -69,7 +69,13 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
   @Override
   public void initialize(BatchRuntimeContext context) throws Exception {
     super.initialize(context);
-    Schema outputSchema = Schema.parseJson(tableSinkConfig.getSchemaStr());
+    Schema outputSchema = null;
```
Comment (bdmogal) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java:72]: any way this code can be refactored to a separate class? seems to be repeated

File: hydrator-common/src/main/java/co/cask/hydrator/common/RecordPutTransformer.java [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+import co.cask.cdap.api.dataset.table.Put;
+import com.google.common.base.Preconditions;
+
+import java.nio.ByteBuffer;
+import javax.annotation.Nullable;
+
+/**
+ * Transforms records into Puts.
+ */
+public class RecordPutTransformer {
```
Comment (bdmogal) [inline hydrator-common/src/main/java/co/cask/hydrator/common/RecordPutTransformer.java:32]: can you add a comment + jira to remove this class?

**Comment by bdmogal** [comment]
LGTM

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java (hunk 1 above)
Comment (anwar6953) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java:72]: I don't think its worth having this small (3-4) line refactored into its own class.

File: hydrator-common/src/main/java/co/cask/hydrator/common/RecordPutTransformer.java (hunk 2 above)
Comment (anwar6953) [inline hydrator-common/src/main/java/co/cask/hydrator/common/RecordPutTransformer.java:32]: Added.
https://issues.cask.co/browse/CDAP-4656

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java (hunk 1 above)
Comment (bdmogal) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/TableSink.java:72]: :)

**Pull request merged**